backend/src/config/options.ts

```diff
@@ -1,17 +1,19 @@
 import { CorsOptions } from "@nestjs/common/interfaces/external/cors-options.interface";
 import { CookieOptions } from "express";
 
+const PRODUCTION = process.env.NEST_ENV === "prod";
```

Comment:
think might be better to use the `FRONTEND_URL` env variable to determine if it's production or not (if `FRONTEND_URL.includes("localhost") ? "dev" : "prod"`)